src/lumigo_tracer/wrappers/http/http_data_classes.py

```diff
@@ -8,13 +8,15 @@ class HttpRequest:
     uri: str
     headers: dict
     body: bytes
+    omit_skip_path: Optional[List[str]]
```

Comment:
We need to keep the path also here because when there is an error response code from an http request, we double the payload size and when we do it we also do `lumigo_dumps`

Reply:
It shouldn't be here 😂  this is not part of the `HttpRequest` 🤷 
WDYT about putting it in the `HttpState`?